File: Java/UberVan.java
package Java;

class UberVan extends Car {
       //marca              Modelo
       Map<String, ArrayList<String,Integer>> typeCarAccepted;
       ArrayList<String> seatsMaterial;

    public UberVan(String license, Account driver, 
    Map<String, ArrayList<String,Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterial)
    {
        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
    
}

File: PHP/UberVan.php
<?php
require_once('car.php');
class UberVan extends Car 
{
    public $typecCarAccepted;
    public $seatsMaterial;


    public function __construct($license, $driver, $typecCarAccepted, $seatsMaterial)
    {
        parent::__construct($license, $driver);
        $this -> typecCarAccepted = $typecCarAccepted;
        $this -> seatsMaterial = $seatsMaterial;
    }
 
}
